### PR TITLE
style: Move `clear` CSS property outside mako

### LINKED ISSUE #21155: style: Move clear outside mako
This is part of #19015.

### PR DESCRIPTION
**Pull request by kleinph**
This is part of issue #19015.

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #21155

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21156)

### CODE REVIEW

File: components/style/values/computed/box.rs [hunk 1]
```diff
@@ -83,3 +83,59 @@ impl ToComputedValue for SpecifiedFloat {
         }
     }
 }
+
+#[allow(missing_docs)]
```
Comment (CYBAI) [inline components/style/values/computed/box.rs]: How about having the `spec url` (https://drafts.csswg.org/css-box/#propdef-clear) for `enum Clear` and removing this `#[allow(missing_docs)]`?

**Review by emilio — APPROVED** [review]
Looks amazing, thanks again for this!

**Comment by kleinph** [comment]
> Looks amazing, thanks again for this!

Thanks again!

Should I add the doc comment for the specified enum?

**Comment by jdm** [comment]
@bors-servo retry
* #20734

**Comment by emilio** [comment]
> Should I add the doc comment for the specified enum?

That'd be nice if you want to! Maybe as a followup PR if CI is fine with this one? :)

**Pull request merged**